docs/contributing.rst

```diff
@@ -1,84 +1,95 @@
-     - Node.js application providing the workflow engine.
-   * - on-dhcp-proxy
-     - https://github.com/RackHD/on-dhcp-proxy
-     - Node.js application providing DHCP proxy support in the workflow engine.
-
-
-
-Libraries
-^^^^^^^^^
-.. list-table::
-   :widths: 20 20 100
-   :header-rows: 1
-
-   * - Library
-     - Repository
-     - Description
-   * - core
-     - https://github.com/RackHD/on-core
-     - Core libraries in use across Node.js applications.
-   * - tasks
-     - https://github.com/RackHD/on-tasks
-     - Node.js task library for the workflow engine.
-
-
-Supplemental code
-^^^^^^^^^^^^^^^^^
-.. list-table::
-   :widths: 20 20 100
-   :header-rows: 1
-
-   * - Library
-     - Repository
-     - Description
-
-   * - Web user interface
-     - | https://github.com/RackHD/on-web-ui
-     - | Initial web interfaces to some of the APIs - multiple
-       | interfaces embedded into a single project.
-   * - statsd
-     - https://github.com/RackHD/on-statsd
-     - | A local statsD implementation that makes it easy to deploy
-       | on a local machine for capturing application metrics.
+We certainly welcome and encourage contributions in the form of issues and pull requests, but please read the
+guidelines in this document before you get involved.
+
+Since our project is relatively new, we don't yet have many hard and fast rules. As the project grows and more
+people get involved, we will solidify and extend our guidelines as needed.
+
+
+
+
+Understand the Repositories
+-------------------------------------------------
+
+The https://github.com/emccode/RackHD repository acts as a single source location to help you get or build all the pieces to learn about, take advantage of, and contribute to RackHD.
+
+A thorough understanding of the individual repositories is essential for contributing to the project. The repositories are described in our documentation at
+http://rackhd.readthedocs.org/en/latest/repositories.html.
+
+
+
+
+
+Submitting Contributions
+----------------------------------------
+
+
+To submit coding additions or changes for a repository, fork the repository and clone it locally. Then use a unique branch to make commits and send pull requests.
+
+Keep your pull requests limited to a single issue. Make sure that the description of the pull request is clear and complete.
+
+Run your changes against existing tests or create new ones if needed. Keep tests as simple as possible.  At a minimum, make sure your changes don’t break the existing project.
+
+After receiving the pull request, our core committers will give you feedback on your work and may request that you make further changes and resubmit the request. The core committers will handle all merges.
+
+If you have questions about the disposition of a  request, feel free to email one of our core committers:
+
+* Ben.BroderickPhillips@emc.com
+* Felix.Yuan@emc.com
+* Joseph.Longever@emc.com
+* Andrew.Hou@emc.com
+* Brian.Parry@emc.com
+* John.Frey@emc.com
```

Comment:
Contrary to what I said earlier, drop John Frey from this list - he'll be transitioning to a different team and duties in short order, so we might drop him from this list now